### PR TITLE
#4271 adding change account button

### LINKED ISSUE #4271: Adding change account button
maybe we can just add change account btn for now?

_Originally posted by @prachi00 in https://github.com/kodadot/nft-gallery/issues/4222#issuecomment-1302838129_

### PR DESCRIPTION
**Pull request by prachi00**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).

👇 \_ Let's make a quick check before the contribution.

## PR Type

- [ ] Bugfix
- [ ] Feature
- [x] Refactoring

## PR Context?

- [x] Closes #4271
- [ ] Requires deployment <>

#### Before submitting pull request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [x] I've posted a screenshot of demonstrated change in this PR

#### Optional

- [ ] I've tested it at </bsx/collection>
- [ ] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [ ] I've found edge cases

#### Had issue bounty label?

- [x] Fill up your KSM address: 
[Payout](https://beta.kodadot.xyz/transfer/?target=EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU)

#### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

## Screenshot 📸

- [x] My fix has changed **something** on UI; a screenshot is best to understand changes for others.
Screen Shot 2022-11-05 at 5 58 56 PM

### CODE REVIEW

**Comment by yangwao** [comment]
@exezbcz I guess we can have much better design for this?

**Comment by prachi00** [comment]
Screen Shot 2022-11-06 at 8 39 41 PM
@yangwao @exezbcz updated the btns according to format, for the rest (asset, balance, usd), I feel like it should be a new issue since it would include logical changes and should be handled in a new issue

**Review by roiLeo — CHANGES_REQUESTED** [review]
- "Change Account" btn is not working on mobile for me

**Comment by prachi00** [comment]
@roiLeo works now

**Review by roiLeo — COMMENTED** [review]
> works now

Capture d’écran 2022-11-09 à 8 33 09 AM

☹️

File: components/rmrk/Profile/NavbarProfileDropdown.vue [hunk 1]
```diff
@@ -293,18 +293,29 @@
...
+          aria-role="menuitem"
+          paddingless
+          class="is-flex is-justify-content-center">
+          <b-button
+            class="navbar__sign-out-button menu-item mb-4 is-size-7"
+            @click="disconnect()">
+            {{ $t('profileMenu.disconnect') }}
+          </b-button>
+        </b-dropdown-item>
+      </div>
```
Comment (roiLeo) [inline components/rmrk/Profile/NavbarProfileDropdown.vue:318]: I would change this code by something more simple

```suggestion
      <b-dropdown-item
        custom
        aria-role="menuitem">
        <div class="buttons is-justify-content-space-between my-2">
          <ConnectWalletButton
          label="general.change_account"
          class="navbar__sign-out-button menu-item is-size-7"
          @closeBurgerMenu="closeBurgerMenu" />
          <b-button
            class="navbar__sign-out-button menu-item is-size-7"
            @click="disconnect()">
            {{ $t('profileMenu.disconnect') }}
          </b-button>
        </div>
      </b-dropdown-item>
    </b-dropdown>
```

File: components/rmrk/Profile/NavbarProfileDropdown.vue [hunk 2]
```diff
@@ -483,12 +494,20 @@ export default class NavbarProfileDropdown extends mixins(
...
     &:hover {
       border: 1px solid $black;
     }
   }
+
+  .dropdown-btns {
+    display: flex;
+    gap: 15px;
+    padding: 16px;
+  }
```
Comment (roiLeo) [inline components/rmrk/Profile/NavbarProfileDropdown.vue:511]: & remove this 
```suggestion
```

File: styles/themes/_dark.scss [hunk 3]
```diff
@@ -60,8 +60,10 @@ html.dark-mode {
     }
 
     .navbar__sign-out-button {
-      border-color: $white;
+      border-color: $white!important;
       box-shadow: 4px 4px 0px $white !important;
+      background: $dark-accent!important;;
```
Comment (roiLeo) [inline styles/themes/_dark.scss:65]:
```suggestion
      background: $dark-accent!important;
```

**Review by roiLeo — COMMENTED** [review]
small notes

**Review by roiLeo — APPROVED** [review]
✅ lgtm, let's finish #4222 too

**Comment by yangwao** [comment]
pay 50

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $50 @ 28.23 USD/KSM ~ 1.771 $KSM
🧗 [EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU](https://kusama.subscan.io/account/EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU)
🔗 [0x51f3429868cdb605018bfd06886e8dd5bff71ae5dcee27766d6739694c49bcae](https://kusama.subscan.io/extrinsic/0x51f3429868cdb605018bfd06886e8dd5bff71ae5dcee27766d6739694c49bcae)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**